package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Sauce Labs Playwright Test Runner",
   "license": "MIT",
   "private": true,
-  "main": "index.js",
+  "main": "src/folio-recorder.js",
```

Comment:
@dpgraham is the plan to cleanup index.js later? or can we remove this file now?

Reply:
There was never an `index.js`. It was the default and didn't do anything. We use this file though.